packages/core/contracts/insured-bridge/ovm/OVM_BridgeDepositBox.sol

```diff
@@ -2,44 +2,16 @@
 pragma solidity ^0.8.0;
 
 import "../BridgeDepositBox.sol";
-
-import "../../external/ovm/OVM_CrossDomainEnabled.sol";
-
-/**
- * @title Lib_PredeployAddresses
- * @dev Copied from https://github.com/ethereum-optimism/optimism/blob/b6fa9e4085da581fb47993b9c895fa20a631897d/packages/contracts/contracts/libraries/constants/Lib_PredeployAddresses.sol
- */
-library Lib_PredeployAddresses {
```

Comment:
we no longer need to define this lib like this and can pull these constants directly from their package.